Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -250,7 +250,18 @@ private extension MoyaProvider {
             let result = convertResponseToResult(response, request: request, data: data, error: error)
             // Inform all plugins about the response
             plugins.forEach { $0.didReceive(result, target: target) }
-            progressCompletion?(ProgressResponse(response: result.value))
+            if let progressCompletion = progressCompletion {
```

Comment:
Do you think we should add tests that check whether the last result have both progress & response?